### PR TITLE
Cloud hosted guide adjustments

### PR DESCRIPTION
**Pull request by AustinSeto**

### CODE REVIEW

**Review by rutavshah — COMMENTED** [review]
Tested and passed on cloud hosted environment and statically.

**Review by rutavshah — APPROVED** [review]
Tested on cloud hosted environment and passed

File: README.adoc [hunk 1]
```diff
@@ -72,21 +73,26 @@ Learn more about containers on the https://www.docker.com/what-container[officia
...
+bundled together with the entire environment that it needs to run.
+By doing this, you know that at any point in your iterative development process,
+the application can run inside that container.
+This helps avoid any unpleasant surprises when you go to test or deploy your application down the road.
+Containers run quickly and do not have a major impact on the speed of your iterative development.
 
 == Additional prerequisites
 
-Before you begin, Docker needs to be installed. For installation instructions, refer to the https://docs.docker.com/get-docker/[official Docker documentation^]. You will build and run the application in Docker containers.
+Before you begin, Docker needs to be installed.
```
Comment (dmuelle) [inline README.adoc]: Before you begin, you need to install Docker if it is not already installed.

File: README.adoc [hunk 2]
```diff
@@ -244,12 +281,27 @@ EndpointIT.java
...
 
+// Static guide version
+ifndef::cloud-hosted[]
 You can test this service manually by starting a server and going to the 
 http://localhost:9080/system/properties-new[http://localhost:9080/system/properties-new^] URL. 
 However, automated tests are a much better approach because they trigger a failure if a change introduces a bug.
 JUnit and the JAX-RS Client API provide a simple environment to test the application. 
-You can write tests for the individual units of code outside of a running application server, or they
-can be written to call the application server directly. In this example, you will create a test that calls the application server directly.
+You can write tests for the individual units of code outside of a running application server,
```
Comment (dmuelle) [inline README.adoc:290]: or they can be written to call the application server directly.
--->
or you can write them to call the application server directly.

File: README.adoc [hunk 3]
```diff
@@ -244,12 +281,27 @@ EndpointIT.java
...
+endif::[]
+
+// Cloud hosted guide version
+ifdef::cloud-hosted[]
+You can test this service manually by starting a server and going to the 
+**http://localhost:9080/system/properties-new** URL.
+However, automated tests are a much better approach because they trigger a failure if a change introduces a bug.
+JUnit and the JAX-RS Client API provide a simple environment to test the application. 
+You can write tests for the individual units of code outside of a running application server,
+or they can be written to call the application server directly.
```
Comment (dmuelle) [inline README.adoc]: see previous comment

File: README.adoc [hunk 4]
```diff
@@ -318,6 +370,12 @@ mvn liberty:devc \
...
 
-To learn more about dev mode with a container and its different
-features, check out the  
-https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#devc-container-mode[Documentation].
+To learn more about dev mode with a container and its different features, 
+check out the http://github.com/OpenLiberty/ci.maven/blob/main/docs/dev.md#devc-container-mode[Documentation^].
+
+== Great work! You're done!
+
+You have just iteratively developed a simple REST application in a container with Open Liberty and Docker.
```
Comment (dmuelle) [inline README.adoc]: You have just
--->
You just

**Review by dmuelle — CHANGES_REQUESTED** [review]
a few edits suggestions

**Comment by AustinSeto** [comment]
Thank you @dmuelle, all suggested changes made

**Review by dmuelle — APPROVED** [review]
LGTM

**Pull request merged**